frontend/workflows/projectSelector/src/project-selector.tsx

```diff
@@ -159,6 +160,7 @@ const ProjectSelector = () => {
     });
 
     // Update!
+    storeState(state);
```

Comment:
should this use `derivedState` instead like the other areas in the hook? played with it locally tho and didn't see any issues

Reply:
nvm using `state` is correct. otherwise if we use derivedState, we don't preserve the checked status of hidden projects.